### PR TITLE
Removing Helm meeting from readme

### PR DESCRIPTION
**Pull request by jmrodri**

### CODE REVIEW

**Review by fabianvf — APPROVED** [review]

**Comment by jmrodri** [comment]
@jberkhahn @asmacdo @joelanford

File: README.md [hunk 1]
```diff
@@ -106,17 +106,6 @@ To organize and steer the development of
...
-### Meetings
-
-* Issue Triage and Community Meeting: Tuesdays (biweekly) at 11:00 Eastern Time. [Convert to your timezone](http://www.thetimezoneconverter.com/?t=11:00&tz=ET%20%28Eastern%20Time%29).
-    * [Meeting Link][meeting-link]: Password: 77777
-    * [Issue Query](https://github.com/operator-framework/helm-operator-plugins/issues?q=is%3Aopen+is%3Aissue+no%3Amilestone+sort%3Acreated-asc)
-    * [Agenda](https://docs.google.com/document/d/112b0ZhSid8exnm72sMT7rDzBEkLlE5s74s0pcn44ssg/edit#)
-
 ### Organizers
 
 * Fabian von Feilitzsch (**[@fabianvf](https://github.com/fabianvf)**), Red Hat
```
Comment (joelanford) [inline README.md:111]: Looks like the `Organizers` and `Contact` sections should also be removed.

Did we check with Simon about this?

Comment (jmrodri) [reply README.md]: Updated. No one was coming to the meetings for months. We even changed them to monthly.

**Review by asmacdo — APPROVED** [review]
/approve

**Pull request merged**